course-v2/layouts/partials/resources_header.html

```diff
@@ -7,14 +7,14 @@
   <h2>Download</h2>
   <div class="hide-offline download-course-container background-color-light-grey p-3">
     <div class="row">
-      <div class="d-flex justify-content-center col-3 px-3">
+      <div class="d-flex justify-content-center col-12 col-md-3 px-3 pb-2 pb-md-0">
         <a class="download-course-button p-2" href="{{ $downloadCourseUrl }}">
           <span class="material-icons">file_download</span> Download course
         </a>
       </div>
-      <div class="col-9 pl-0 pr-3">
-        This package contains the same content as the online version of the course, 
-        <em>except for the audio/video materials</em>. These can be downloaded below. For help 
+      <div class="col-12 col-md-9 pr-3">
```

Comment:
This looks good to me. My only comment is that the `pr-3` classes don't seem necessary to me; bootstrap columns include padding by default.

(Looking into it out of curiosity... the default padding is 1rem (=font size on root element) which on our site is 14px; pr-3 is 15px, so I guess this gives us 1 extra pixel on the right-hand side. I'd go with less css.)

Anyway, that wasn't added in this PR, so probably not worth changing, but wanted to mention it since we were talking about css stuff this morning.

Reply:
Updated in 08197e3ccb7caf27b3d41719c0d703515d5ba2b3.